web/src/views/list-happenings.js

```diff
@@ -50,8 +59,10 @@ function HappeningTableHeader({onClick}) {
   return <thead>
     <th>Name</th>
     <th>Description</th>
+    <th></th>
     <th>
```

Comment:
maybe we could use colspan here if it makes sense

Reply:
colspan makes create button appear on top of details button instead of delete

Reply:
Never mind. I've fixed that